_posts/2020-10-12-sparklyr-flint-0.2.0-released/sparklyr-flint-0.2.0.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+title: "sparklyr.flint 0.2: ASOF Joins, OLS Regression, and Additional Summarizers"
+description: |
+  We are excited to report a number of powerful new functionalities and improvements which are now part of sparklyr.flint 0.2!
+author:
+  - name: Yitao Li
+    url: https://github.com/yitao-li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+slug: sparklyr.flint-0.2.0
+date: 10-12-2020
+categories:
+  - R
+  - Packages/Releases
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/sparklyr-flint-0.2.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+After [`sparklyr.flint`](https://cran.r-project.org/web/packages/sparklyr.flint/index.html), a [`sparklyr`](https://sparklyr.ai) extension for leveraging [`Flint`](https://github.com/twosigma/flint) time series functionalities through `sparklyr`, has been [introduced](https://blogs.rstudio.com/ai/posts/2020-09-07-sparklyr-flint) in September, we have made a number of enhancements to it, and have successfully submitted `sparklyr.flint` 0.2 to CRAN recently.
```

Comment:
```suggestion
Since [`sparklyr.flint`](https://cran.r-project.org/web/packages/sparklyr.flint/index.html), a [`sparklyr`](https://sparklyr.ai) extension for leveraging [`Flint`](https://github.com/twosigma/flint) time series functionalities through `sparklyr`, was [introduced](https://blogs.rstudio.com/ai/posts/2020-09-07-sparklyr-flint) in September, we have made a number of enhancements to it, and have successfully submitted `sparklyr.flint` 0.2 to CRAN recently.
```